source/CardGames.py

```diff
@@ -156,18 +161,24 @@ def setHand(self, cards: "list[Card]", isKnown: bool = False):
     self.hand = cards
     self.knownCards = [isKnown for _ in self.hand]
 
-  def showHand(self, printShort: bool = False):
+  def showHand(self, printShort: bool = False, showBack: bool = False):
     for idx in range(6):
       for i, card in enumerate(self.hand):
```

Comment:
Having idx and i in the same loop is confusing to me since i stands for index. Usually I use i and j